test/Shared.ps1

```diff
@@ -73,6 +76,8 @@ function NewGitTempRepo([switch]$MakeInitialCommit) {
     Set-Location $repoPath
 
     if ($MakeInitialCommit) {
+        git config user.email "spaceman.spiff@appveyor.com"
+        git config user.name "Spaceman Spiff"
```

Comment:
Should this not use `$gitbin`?

Reply:
Yup.  Sorry about that. Guess I was in a bit too much of hurry on this PR.  New PR coming.